sbncode/FluxReader/job/fluxfilter.fcl

```diff
@@ -0,0 +1,12 @@
+# Configuration fhicl file for filter module FluxGeoFilter
+
+BEGIN_PROLOG
+flux_filter: {
+  module_type: "FluxGeoFilter"
+
+  # volWorld, volDetEnclosure, volCryostat, volTPC, volTPCActive, ...
+  volumes: [ "volTPCActive" ]
```

Comment:
This only works for SBND, correct? Should we label it as such in the fcl name "_sbnd.fcl"?

Reply:
Yes, correct. Done with commits 60dcb63, fa717c8.